composer.json

```diff
@@ -15,7 +15,9 @@
         "ext-readline": "*",
         "ext-redis": "*",
         "galbar/jsonpath": "^1.3.1",
-        "league/flysystem": "^2.5.0",
+        "league/flysystem": "^3.7.0",
+        "league/flysystem-ftp": "^3.25.0",
+        "league/flysystem-local": "^3.25.0",
```

Comment:
We already have those dependencies as part of core dependencies for 
```
        "spryker/flysystem": "^3.0.0",
        "spryker/flysystem-ftp-file-system": "^3.0.0",
        "spryker/flysystem-local-file-system": "^3.0.0",
```

Please remove